extensions/sql-database-projects/src/typings/vscode-mssql.d.ts

```diff
@@ -5,613 +5,638 @@
-        ownerUri: string;
-        taskExecutionMode: TaskExecutionMode;
-    }
-
-    export interface GenerateDeployPlanParams {
-        packageFilePath: string;
-        databaseName: string;
-        ownerUri: string;
-        taskExecutionMode: TaskExecutionMode;
-    }
-
-    export interface GetOptionsFromProfileParams {
-        profilePath: string;
-    }
-
-    export interface ValidateStreamingJobParams {
-        packageFilePath: string;
-        createStreamingJobTsql: string;
-    }
-
-    export interface SchemaCompareGetOptionsParams { }
-
-    export interface SchemaCompareOptionsResult extends ResultStatus {
-        defaultDeploymentOptions: DeploymentOptions;
-    }
-
-    export interface ITreeNodeInfo extends vscode.TreeItem {
-        readonly connectionInfo: IConnectionInfo;
-        nodeType: string;
-        metadata: ObjectMetadata;
-        parentNode: ITreeNodeInfo;
-    }
-
-    export const enum MetadataType {
-        Table = 0,
-        View = 1,
-        SProc = 2,
-        Function = 3
-    }
-
-    export interface ObjectMetadata {
-        metadataType: MetadataType;
-
-        metadataTypeName: string;
-
-        urn: string;
-
-        name: string;
-
-        schema: string;
-    }
-
-    /**
-     * Azure functions binding type
-     */
-    export const enum BindingType {
-        input,
-        output
-    }
-
-    /**
-     * Parameters for adding a SQL binding to an Azure function
-     */
-     export interface AddSqlBindingParams {
-        /**
-         * Aboslute file path of file to add SQL binding
-         */
-        filePath: string;
-
-        /**
-         * Name of function to add SQL binding
-         */
-        functionName: string;
-
-        /**
-         * Name of object to use in SQL binding
-         */
-        objectName: string;
-
-        /**
-         * Type of Azure function binding
-         */
-        bindingType: BindingType;
-
-        /**
-         * Name of SQL connection string setting specified in local.settings.json
-         */
-        connectionStringSetting: string;
-    }
-
-    /**
-     * Parameters for getting the names of the Azure functions in a file
-     */
-    export interface GetAzureFunctionsParams {
-        /**
-         * Absolute file path of file to get Azure functions
-         */
-        filePath: string;
-    }
-
-    /**
-     * Result from a get Azure functions request
-     */
-    export interface GetAzureFunctionsResult extends ResultStatus {
-        /**
-         * Array of names of Azure functions in the file
-         */
-        azureFunctions: string[];
-    }
+	import * as vscode from 'vscode';
+
+	/**
+	 * Covers defining what the vscode-mssql extension exports to other extensions
+	 *
+	 * IMPORTANT: THIS IS NOT A HARD DEFINITION unlike vscode; therefore no enums or classes should be defined here
+	 * (const enums get evaluated when typescript -> javascript so those are fine)
+	 */
+
+
+	export const enum extension {
```

Comment:
A lot of other lines besides got changed that shouldn't have been updated with these changes. Can you undo the whitespace changes in this file?

Reply:
@kisantia Just as an FYI, you can hide whitespace changes too in the diff by clicking the gear icon at the top and selecting "hide whitespace" 

![image](https://user-images.githubusercontent.com/28519865/149017180-0a0673f6-8d11-4347-88cd-1ca4894e460c.png)

But in this case @ssreerama since this file is coming from the VS Code extension and it looks like the formatting is different there it would be best to make any fixes/changes/formatting there and then just copy the file over directly instead of formatting it here since they apparently have different formatting rules. (which is something we should align to at some point I'd think but that's a future task..)

Reply:
I agree! It was completely my mistake as I did hygiene check for VsCode file too like we do for our own files, which made these unwanted changes, reverted them. Thanks for the information @Charles-Gagnon